dsc/src/util.rs

```diff
@@ -39,7 +41,7 @@ use syntect::{
     parsing::SyntaxSet,
     util::{as_24_bit_terminal_escaped, LinesWithEndings}
 };
-use tracing::{Level, debug, error, warn, trace};
+use tracing::{Level, info, debug, error, warn, trace};
```

Comment:
Can you fix this so it's in alpha order?

Reply:
fixed.